src/main/java/org/folio/circulation/domain/representations/ItemReportRepresentation.java

```diff
@@ -60,9 +61,24 @@ public JsonObject createItemReport(InTransitReportEntry inTransitReportEntry) {
     if (loan != null) {
       writeLoan(itemReport, loan);
     }
+
+    final LastCheckIn lastCheckIn = item.getLastCheckIn();
+    if (lastCheckIn != null) {
+      writeLastCheckIn(itemReport, lastCheckIn);
```

Comment:
This suggests that new properties have been added to the item representation in the report. If so, please can the interface version be updated (in the descriptor and RAML file) and the relevant schema updated.

Reply:
Updated